scrapers/infernalrestraints.yml

```diff
@@ -0,0 +1,40 @@
+name: infernalrestraints
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - infernalrestraints.com/device
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //a[@class="articleTitleText"][1]/text()
+      Date:
+        selector: //span[@class="articlePostDateText"][1]/text()
+        postProcess:
+          - parseDate: "January 02, 2006"
+          - parseDate: "January 2, 2006"
+      Details:
+        selector: //td[@class="articleCopyText"]/text()
```

Comment:
the <br> 's in the details are already parsed so you shouldn't need to do all that
```yaml
      Details:
        selector: //td[@class="articleCopyText"]/text()
        concat: "\n\n"
```
should be enough

Reply:
fixed